doc/index.rst

```diff
@@ -173,6 +173,7 @@ Note that some of these currently tied only to a single option, but we still lea
 * **CATKIN_LINT** (default: not set. Value range: [true|pedantic]): If ``true``, run `catkin_lint <http://fkie.github.io/catkin_lint/>`__ with ``--explain`` option. If ``pedantic``, ``catkin_lint`` command runs with ``--strict -W2`` option, i.e. more verbose output will print, and the CI job fails if there's any error and/or warning occurs. Industrial CI uses the `latest version available from pypi <https://pypi.org/project/catkin-lint/>`__. If the older version in the `ros repository <http://packages.ros.org/ros/ubuntu/pool/main/c/catkin-lint/>`__ is required, :code:`ADDITIONAL_DEBS='python-catkin-lint'` can be added to the CI Config.
 * **CATKIN_LINT_ARGS** (default: not set): If true, you can pass whatever argument(s) ``catkin_lint`` takes, except ``--explain`` that is set by default. Options can be delimit by space if passing multiple.
 * **CMAKE_ARGS** (default: not set): CMake arguments that get passed to the builder for all workspaces.
+* **CC** / **CXX** (default: not set): Environment variables to specify the C/C++ compilers. If required, these can be installed by specifying `ADDITIONAL_DEBS`. E.g. `ADDITIONAL_DEBS=clang CC=clang CXX=clang++` uses Clang to build the workspaces.
```

Comment:
Would perhaps be good to mention these aren't specific to `industrial_ci`, but the regular env vars you'd use to specify a compiler to use when invoking CMake?

Reply:
Good call - I've just amended. Please let me know if you'd like me to use a different formulation or move it to a separate section